src/api/goods.ts

```diff
@@ -10,10 +10,12 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods.sort(
+      (goods1, goods2) => goods1.name.localeCompare(goods2.name),
```

Comment:
Consider renaming the variables 'goods1' and 'goods2' to 'good1' and 'good2' for better readability and to accurately represent the individual items being compared.